src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -1,28 +1,33 @@
 namespace FSharp.Control
 
+open System.Collections.Generic
+open System.Threading.Tasks
+
 #nowarn "1204"
 
-module TaskSeq =
-    open System.Collections.Generic
-    open System.Threading.Tasks
+[<AutoOpen>]
+module TaskSeqExtensions =
+    module TaskSeq =
+        /// Initialize an empty task sequence.
+        val empty<'T> : taskSeq<'T>
 
-    /// Initialize an empty taskSeq.
-    val empty<'T> : taskSeq<'T>
+[<Sealed; AbstractClass>]
+type TaskSeq =
 
     /// <summary>
     /// Creates a <see cref="taskSeq" /> sequence from <paramref name="source" /> that generates a single element and then ends.
     /// </summary>
     ///
     /// <param name="value">The input item to use as the single item of the task sequence.</param>
-    val singleton: value: 'T -> taskSeq<'T>
+    static member singleton: value: 'T -> taskSeq<'T>
```

Comment:
so tempting to review all the xmldoc in here but best to allow this PR to say focused!

Reply:
Haha, yeah. There's definitely still more fine tuning needed in these doc comments (which, tbh, are more tailored to being well suited towards tooltip comments, as I am currently not outputing any documentation, but I should do that soon as well, though).

Reply:
Just for this one, I made an adjustment, though.

Reply:
That's going to be an crazy amount of work (and that's only thinking from the reviewing side!)
Side note: for 187's overview, you've swapped CamelCase and pascalCase ;)

Reply:
Fixed 👍